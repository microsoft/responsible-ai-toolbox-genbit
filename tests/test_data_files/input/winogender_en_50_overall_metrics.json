{
  "DataSet": "winogender-schemas-master",
  "Freq_cutoff": 23.543112000000008,
  "Num_words_considered": 73,
  "Freq_of_male_gendered_words": 592,
  "Freq_of_female_gendered_words": 244,
  "Percentage_of_female_gendered_words": 0.291866028708134,
  "Percentage_of_male_gendered_words": 0.708133971291866,
  "JSD": 0.005559924571846245,
  "Avg_bias_ratio": 0.8826290307923338,
  "Avg_bias_conditional": 0.4330195577582524,
  "Avg_bias_ratio_absolute": 0.8826290307923338,
  "Avg_bias_conditional_absolute": 0.44533762551261086,
  "Std_dev_bias_ratio": 0.1790225045866468,
  "Std_dev_bias_conditional": 0.1790225045866468,
  "bias_percentage": 160,
  "language": "en",
  "context_window": 5,
  "distance_weight": 0.95,
  "percentile_cutoff": 80,
  "tokenizer": "stanza",
  "matched_tokens": {
    "male_tokens": "['he', 'his', 'him']",
    "female_tokens": "['she', 'her']"
  }
}