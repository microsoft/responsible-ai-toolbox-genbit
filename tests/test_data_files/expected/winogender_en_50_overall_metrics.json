{"genbit_score": 0.44533762551261086, "percentage_of_female_gender_definition_words": 0.291866028708134, "percentage_of_male_gender_definition_words": 0.7081339712918661, "percentage_of_non_binary_gender_definition_words": 0.0, "percentage_of_trans_gender_definition_words": 0.0, "percentage_of_cis_gender_definition_words": 1.0, "additional_metrics": {"avg_bias_ratio": 0.8826290307923338, "avg_bias_conditional": 0.4330195577582524, "avg_bias_ratio_absolute": 0.8826290307923338, "avg_bias_conditional_absolute": 0.44533762551261086, "avg_trans_cis_bias_ratio": 0.0, "avg_trans_cis_bias_conditional": -1.1834382018842289, "avg_trans_cis_bias_ratio_absolute": 0.0, "avg_trans_cis_bias_conditional_absolute": 1.1834382018842289, "std_dev_bias_ratio": 0, "std_dev_bias_conditional": 0, "std_dev_trans_cis_bias_ratio": 0, "std_dev_trans_cis_bias_conditional": 0}}
